packages/foreman/rubygem-nio4r/rubygem-nio4r.spec

```diff
@@ -52,6 +52,9 @@ gem spec %{SOURCE0} -l --ruby > %{gem_name}.gemspec
 %{?scl:EOF}
 
 %build
+# Use -fno-strict-alisaing, due to various "warning: dereferencing type-punned pointer will break strict-aliasing rules"
+export CONFIGURE_ARGS="--with-cflags='%{optflags} -fno-strict-aliasing'"
```

Comment:
This is what Fedora uses:
```suggestion
# Update %%optflags used in %%gem_install to avoid strict-aliasing warnings.
# https://github.com/socketry/nio4r/pull/130
# http://pkgs.fedoraproject.org/cgit/rpms/rubygems.git/tree/macros.rubygems
%global optflags %{?optflags} -fno-strict-aliasing
```

Reply:
I'll test fedora's syntax in a scratch build real quick.

My fix was based off of:
https://fedoraproject.org/wiki/Archive:PackagingDrafts/RubyGem_with_C_code#.5BSOLVED.5D_To_pass_CFLAGS_correctly

And pre-existing examples in packaging.

Reply:
confirmed fedora syntax worked as intended, updated to that syntax.